src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -681,6 +687,8 @@ private static class ChannelHolder {
         private Channel channel;
         @GuardedBy("lock")
         private boolean isSubscribed = false;
+        @GuardedBy("lock")
+        private boolean isSubscribing = false;
```

Comment:
Maybe we should replace this and `isSubscribed` boolean variables with enum that will have channel states